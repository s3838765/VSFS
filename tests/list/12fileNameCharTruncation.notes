NOTES V1.0
@This-is-a-very-long-file-name-that-will-exceed-255-characters-and-it-should-truncate-the-line-at-the-very-end-if-this-works-correctly_abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ_1234567890_255-characters-is-a-lot-of-characters-for-a-file-name_the-file-extension-will-stay-on-the-file-but-the-main-file-name-will-be-truncated.txt
