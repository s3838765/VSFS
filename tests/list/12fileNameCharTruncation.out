-rwxrwxrwx 1 dib dib 0 Oct 22 17:48 This-is-a-very-long-file-name-that-will-exceed-255-characters-and-it-should-truncate-the-line-at-the-very-end-if-this-works-correctly_abcdefghijklmnopqrstuvwxyz_ABCDEFGHIJKLMNOPQRSTUVWXYZ_1234567890_255-characters-is-a-lot-of-characters-for-a-file-na.txt
